# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: modules/libpref/init/StaticPrefList.yaml
# Commit: d393067941bd
# Full Hash: d393067941bd69dfa7eff7790e19a0a0162076cd
# Author: Tim Huang <tihuang@mozilla.com>
# Date: 2022-07-20 09:59:35
# Regressor Bug: 1773701
# File Overlap Count: 1
# Description:
#   Bug 1773701 - Part 2: Implement the email tracking data collection feature. r=dimi
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D151522
# ==============================================================================

diff -r f3c636e27b0b -r d393067941bd modules/libpref/init/StaticPrefList.yaml
--- a/modules/libpref/init/StaticPrefList.yaml	Tue Jul 19 20:04:24 2022 +0000
+++ b/modules/libpref/init/StaticPrefList.yaml	Tue Jul 19 20:04:24 2022 +0000
@@ -11725,6 +11725,12 @@
   value: @IS_NIGHTLY_BUILD@
   mirror: always
 
+# Collecting 3rd party emailtracking telemetry.
+- name: privacy.trackingprotection.emailtracking.data_collection.enabled
+  type: bool
+  value: true
+  mirror: always
+
 # Whether Origin Telemetry should be enabled.
 # NOTE: if telemetry.origin_telemetry_test_mode.enabled is enabled, this pref
 #       won't have any effect.